### DISCUSSION TITLE
Still use low `CPU` even if the control does not refresh

### DISCUSSION THREAD

**Discussion by Coloryr** [AvaloniaUI/Avalonia#12777] [Q&A]
Open a window, and no click it anymore, it still use CPU.
![image](https://github.com/AvaloniaUI/Avalonia/assets/26276037/2fba4e0b-0729-42f0-952f-3404570cf4c0)

![image](https://github.com/AvaloniaUI/Avalonia/assets/26276037/625cef96-4c01-44cb-96d0-4838e4c8c60a)
![image](https://github.com/AvaloniaUI/Avalonia/assets/26276037/17874076-674e-4939-8a8e-11eda3777238)

**Comment by Coloryr** [comment]
Let it behide other windows
![image](https://github.com/AvaloniaUI/Avalonia/assets/26276037/84db7178-1a26-43f2-889f-269a0f4c87df)

Still use CPU

**Comment by Coloryr** [comment]
Minimize window
![image](https://github.com/AvaloniaUI/Avalonia/assets/26276037/08ff38f7-d875-48e7-9ca5-6b85d08907a7)

Use 0.7-1.4%

**Comment by Coloryr** [comment]
![image](https://github.com/AvaloniaUI/Avalonia/assets/26276037/8a6b0cd1-4423-4893-b573-8b62d3c42ffb)

**Comment by Coloryr** [comment]
Can let it not usage CPU when it not refresh?

**Comment by Coloryr** [comment]
![image](https://github.com/AvaloniaUI/Avalonia/assets/26276037/937cbd2a-39bd-4ae8-9b34-1474f7ebd126)

**Reply by rabbitism** [reply]
Consider monitor performance with released app.

**Reply by Coloryr** [reply]
![image](https://github.com/AvaloniaUI/Avalonia/assets/26276037/2c78c739-e301-43a1-93ba-6db57f223a03)

public released

**Comment by Coloryr** [comment]
I founded issues.

When the ProgressBar property set `IsIndeterminate=true`,

```
<ProgressBar
    Height="20"
    Margin="10"
    IsIndeterminate="true" />
```

Even if it's already hidden

```
<ProgressBar
    Height="20"
    Margin="10"
   IsVisible="false"
    IsIndeterminate="true" />
```

**Comment by kyuranger** [comment]
Btw, did you make an Avalonia version of Minecraft?

**Reply by Coloryr** [reply]
What? it's Minecraft launcher.
https://github.com/Coloryr/ColorMC

**Comment by Tamnac** [comment]
I still see something like this, it's much lower, like 0.5%, but still, I don't see why a completely unused application should be touching the cpu
